### ISSUE TITLE
bash script "1Genome_construct_Pangenome.sh" not found 

### ISSUE THREAD

**Issue by bioinfoanalyst** [wjian8/psvcp_v1.01#16]
I tried to run following given command:

`bash $path_of_the_pipeline/1Genome_construct_Pangenome.sh ref.fa query.fa > job.sh && bash job.sh
`
But 1Genome_construct_Pangenome.sh script is not provided in the pipeline kindly update

**Comment by wjian8** [comment]
please use "Refgenome_update_by_quest.sh"
I have updated README. 

`bash $path_of_the_pipeline/Refgenome_update_by_quest.sh ref.fa query.fa > job.sh && bash job.sh`

**Comment by bioinfoanalyst** [comment]
Okay thanks.
Also please suggest if I need to do something with Assemblytics because it does not generate **.bed.Assemblytics_structural_variants.bed** file when running **Construct_pan_and_Call_sv.py**. 

The progress.log file contains following error:

**UNIQFILTER,FAIL,Step 1: Assemblytics_uniq_anchor.py failed: Possible problem with Python or Python packages on server.**

**Comment by bioinfoanalyst** [comment]
By running suggested command I got following error @wjian8 : 

Error in file(file, "rt") : cannot open the connection
Calls: read.table -> file
In addition: Warning message:
In file(file, "rt") :
  cannot open file 'genome_gff_dir_example/MSU_0-2M.pav.gff': No such file or directory
Execution halted

**Comment by wjian8** [comment]
`bash $path_of_the_pipeline/Refgenome_update_by_quest.sh ref.fa query.fa > job.sh `

Did you get the job.sh script? You can use cat job.sh to view its contents. The job.sh script executes commands sequentially. Ensure that the input and output files are specified correctly, as this will help identify where the issue begins.

Have you confirmed that all dependencies are installed?

MUMMER v4.0.0 (https://github.com/mummer4/mummer)

Assemblytics v1.2.1 (https://github.com/marianattestad/assemblytics)

BWA-MEM v0.7.17-r1198-dirty (http://bio-bwa.sourceforge.net/)

Picard (https://github.com/broadinstitute/picard)

Samtools v.1.9-49-gb321ed1 (https://github.com/samtools/samtools)

bedtools v2.29.2 (https://github.com/arq5x/bedtools2)

seqkit v1.2 (https://bioinf.shenwei.me/seqkit/)

blastn v2.2.3 (https://blast.ncbi.nlm.nih.gov/Blast.cgi)

R v3.5.0 or later (https://www.r-project.org/) and packages: -hash -parallel -snowfall

python3 (https://www.python.org/) and packages: -pandas -gzip -argparse -threading -re

**Comment by bioinfoanalyst** [comment]
Yes dependencies are already satisfied. And working on example data as it is.

**Comment by 94yangll** [comment]
> Also by running **Construct_pan_and_Call_sv.py** on example data as given in readme, I got following console:
> 
> `No gene and PAV has overlap, next round`
> 
> Why it is so? Even though it is an example data.

I had the same problem.
@wjian8
